src/utils/text.ts

```diff
@@ -17,10 +17,14 @@ export function expandTabs(line: string, tabWidth: number): string {
     return expanded;
 }
 
+/**
+ * Note: This function is for use by the highlight module only.
+ */
 export function calculateEditorColFromVimScreenCol(line: string, screenCol: number, tabSize: number): number {
```

Comment:
off topic: this could be named `getColFromNvimCol` without any loss of clarity, while reducing verbosity.